src/views/content/markup.tsx

```diff
@@ -50,6 +50,9 @@ export type MarkupDeps = {
 export interface MarkupOptions {
     setPageTitle?: boolean;
 }
+export interface ExtendedMarkupOptions extends MarkupOptions {
```

Comment:
Let's move this to the top of the test file since it isn't needed for any production code.

Reply:
Updated.